src/OrchardCore.Cms.Web/OrchardCore.Cms.Web.csproj

```diff
@@ -11,7 +11,13 @@
     <WarningsNotAsErrors>612,618</WarningsNotAsErrors>
     <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
     <RazorRuntimeCompilation>false</RazorRuntimeCompilation>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings Condition="'$(RazorRuntimeCompilation)' == 'false'">enable</ImplicitUsings>
```

Comment:
Why doesn't the template have this section?

Reply:
`RazorRuntimeCompilation` is OC's custom property and I guess it's used only `OrchardCore.Cms.Web`